test/routes/index-test.js

```diff
@@ -13,10 +13,12 @@ describe('Index page', () => {
         await app.close();
     });
 
```

Comment:
I think we should leave the test for the '/' endpoint as well, only change asserts.

Reply:
As a general rule - avoid removing or altering too much already written tests.

Reply:
I'll use snapshots for that problem. Is it appropriate idea?